ugvc/pipelines/training_prep_pipeline.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
+# Copyright 2022 Ultima Genomics Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+# DESCRIPTION
+#    Prepare data for training
+
+from __future__ import annotations
+
+import argparse
+import logging
+import sys
+
+from ugvc import logger
+from ugvc.filtering import tprep_constants, training_prep
+
+
+def parse_args(argv: list[str]) -> argparse.Namespace:
+    ap_var = argparse.ArgumentParser(prog="training_prep_pipeline.py", description="Prepare training data")
+    ap_var.add_argument("--call_vcf", help="Call VCF file", type=str, required=True)
+    ap_var.add_argument(
+        "--gt_type",
+        help="Ground truth type: exact or approximate",
+        type=tprep_constants.GtType,
+        required=True,
+        default=tprep_constants.GtType.EXACT,
+    )
+    ap_var.add_argument("--blacklist", help="Blacklist file", type=str, required=False)
+    ap_var.add_argument(
+        "--base_vcf", help="Truth VCF file, needed only if gt_type is 'exact'", type=str, required=False
+    )
+    ap_var.add_argument(
+        "--reference", help="Reference file prefix. Requires .fai,and .sdf folder", type=str, required=False
+    )
+    ap_var.add_argument("--hcr", help="High confidence regions BED file", type=str, required=False)
+    ap_var.add_argument(
+        "--custom_annotations",
+        help="Custom INFO annotations to read from the VCF (multiple possible)",
+        required=False,
+        type=str,
+        default=None,
+        action="append",
+    )
+
+    ap_var.add_argument(
+        "--contigs_to_read",
+        help="List of chromosomes to read the data from, default: all",
+        nargs="+",
+        type=str,
+        required=False,
+    )
+    ap_var.add_argument(
+        "--contig_for_test",
+        help="Chromosome to split into test set, **contained** in contigs_to_read",
+        type=str,
+        required=False,
+    )
+    ap_var.add_argument("--output_prefix", help="Output HDF5 files prefix", type=str, required=True)
```

Comment:
I got many outputs in currdir, and on in output_prefix dir, might be nice to change that

Reply:
I ran with --output_prefix ../out_dir

Reply:
--ouptut_prefix includes the beginning of the file name for the output (.h5) is appended to it. Should I change the name or add a parameter for the output directory?

Reply:
@doron-st - sorry - I still can't reproduce this. When I run it from a different directory, I only get files under the output_prefix